docs/manual/manual_buzzer_control.rst

```diff
@@ -0,0 +1,183 @@
+Here, we will discuss all possible “songs” your module may play during normal operation and their meanings. We will also discuss how to use the 
+:ref:`buzzer control client <buzzer_control>` in order to create and play your own songs.
+
+Standard Songs
+==================
+
+Startup
+-----------
+On a successful power-up and initialization, your module will play a 5-tone startup song. After this song completes, your module is ready for use.
+
+The startup song is played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/startup_song.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+Arming
+-------
+If you are using the :ref:`Advanced Arming feature <manual_advanced_arming>`, then, on arming, your module will perform its :ref:`advanced_arming_behavior` which includes the arming song played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/arming_song.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+Disarming
+------------
+If you are using the :ref:`Advanced Arming feature <manual_advanced_arming>`, then, on disarming, your module will perform its :ref:`advanced_disarming_behavior`. 
+If the song playback option is configured to either Once or Continuously, you will hear the disarming song played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/disarm.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+If the song playback option is set to Continuously, the disarm song will repeat until the disarm state is broken.
+
+Analog Hobby Protocol Calibration
+-------------------------------------
+While :ref:`performing analog hobby protocol calibration <hobby_calibration_tutorial>`, you will hear two separate song portions indicating the calibration's progress. 
+The first portion of the song indicates that calibration is starting. This is followed by tones played twice as fast indicating that calibration is in progress. 
+An example of a full calibration song is played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/hobby_calibration.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+Timeout
+----------
+When a :ref:`timeout <manual_timeout>` is reached, your module will play the timeout song as specified by its :ref:`timeout_behavior`. 
+When the Song Playback is set to play either once or continuously, you will here the timeout song played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/timeout.mp3" type="audio/mpeg">
```

Comment:
At the risk of being extremely annoying, I feel like there is something pretty prominent in the background of the audio on this one, like a loud car or a lawn mower or something? If its possible to get a cleaner sample of this, that would be good